manage.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import os
+import sys

```

Comment:
create a .gitignore file and remove all binary files from git including *.pyc..  you can get prepared gitignores from here: https://github.com/github/gitignore/blob/master/Python.gitignore